package.json

```diff
@@ -41,7 +41,7 @@
     "chai": "^4.3.7",
     "esbuild": "^0.17.4",
     "events": "^3.3.0",
-    "maplibre-gl": "^2.1.9",
+    "maplibre-gl": "^2.4.0",
```

Comment:
This is fine, though it should be unnecessary as long as the package isn’t breaking backwards compatibility within the v2._x_ series. The `^` means NPM would resolve to the latest available version before v3.0.0.